src/GraphQL.ApiTests/net6+net7/GraphQL.approved.txt

```diff
@@ -1177,7 +1177,6 @@ namespace GraphQL.Execution
     public class ExecutionStrategyRegistration : System.IEquatable<GraphQL.Execution.ExecutionStrategyRegistration>
     {
         public ExecutionStrategyRegistration(GraphQL.Execution.IExecutionStrategy Strategy, GraphQLParser.AST.OperationType Operation) { }
-        protected virtual System.Type EqualityContract { get; }
```

Comment:
Need advice with this change

Reply:
@Shane32  I think there are no problems if we all move to > 6.0.402 SDK . @zcsizmadia Please rebase on `master`. I removed these lines in #3380 .